lale/lib/lale/map.py

```diff
@@ -29,6 +29,8 @@ def transform(self, X):
         def get_map_function_output(column, new_column_name):
             functions_module = importlib.import_module("lale.lib.lale.functions")
             function_name = column._expr.func.id
+            if _is_ast_attribute(column):
```

Comment:
also, what is the expected behavior of 
`columns=[rename(it.old)]`
`columns=[it.old]`.
We may make sure they do something reasonable and add a test for them.